README.md

```diff
@@ -83,8 +83,32 @@ It is recommended that sensitive data (like the Kubernetes admin password) is st
 
 A CLI is also provided for `fabrik8` that allows you to invoke the API from the command line:
 
+### `fabrik8 create [--name name] [--url url] --spec ./path/to/spec`
+
+Creates a full cluster, reading defaults and existing configuration securely from centralized cluster-info.  The only options that are required are configuration for cluster-info, a name or cluster url, and the path to a McGonagall specification.  If re-running, only a name is required -- options will be re-read from cluster-info.
+
+* `--url`, `-u` the url of the cluster you wish to create,
```

Comment:
nit: an example value here would be nice, url is a little vague (does it include the registry prefix, the www prefix, the protocol, the trailing slash, etc etc)